server/app/services/settings/SettingsManifest.java

```diff
@@ -0,0 +1,1344 @@
+/**
+ * DO NOT EDIT THIS FILE DIRECTLY
+ *
+ * <p>This file is generated based on server/conf/env_var_docs.json
+ *
+ * <p>To update it, run bin/generate-settings-manifest
+ */
+package services.settings;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.typesafe.config.Config;
+import javax.inject.Inject;
+
+/** Data class providing access to server settings. */
+public final class SettingsManifest extends AbstractSettingsManifest {
+
+  private final ImmutableMap<String, SettingsSection> settingsSections;
+
+  @Inject
+  public SettingsManifest(Config config) {
+    super(config);
+    this.settingsSections = GENERATED_SECTIONS;
+  }
+
+  @VisibleForTesting
+  public SettingsManifest(ImmutableMap<String, SettingsSection> settingsSections, Config config) {
+    super(config);
+    this.settingsSections = checkNotNull(settingsSections);
+  }
+
+  /** A map of setting section names to {@link SettingsSection}s. */
+  @Override
+  public ImmutableMap<String, SettingsSection> getSections() {
+    return settingsSections;
+  }
+
+  /** Small logo for the civic entity used on the login page. */
+  public String getWhitelabelSmallLogoUrl() {
+    return getString("WHITELABEL_SMALL_LOGO_URL");
+  }
+
+  /** Logo with civic entity name used on the applicant-facing program index page. */
+  public String getWhitelabelLogoWithNameUrl() {
+    return getString("WHITELABEL_LOGO_WITH_NAME_URL");
```

Comment:
nitpick (non-blocking): regenerate before merging to remove this var that was removed in your other pr.

Reply:
Done